apps/demo/src/app/routing/indicator/indicator.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnInit } from '@angular/core';
+import { Router, RouterEvent, NavigationStart } from '@angular/router';
+import { filter, tap } from 'rxjs/operators';
+import { RouterPercentEnum } from '../router-percent-enum';
+
+
+@Component({
+  selector: 'demo-routing-indicator',
+  templateUrl: './indicator.component.html',
+  styleUrls: ['./indicator.component.scss']
+})
+export class DemoIndicatorComponent implements OnInit {
+  
+  routingPercentage = 0;
+  show = false;
+
+  constructor(private router: Router) { }
+
+  ngOnInit() {
+    this.router.events.pipe(
+      filter(e => e instanceof RouterEvent),
+      filter(e => e.constructor.name in RouterPercentEnum),
+      tap(e => this.routingPercentage = RouterPercentEnum[e.constructor.name]),
+      filter(e => e instanceof NavigationStart),
+      tap(e => this.show = true)
+    ).subscribe();
+  }
+
+  handleAnimationComplete(): void{
```

Comment:
The only thing I don't really like is this name. I think it should be something like `handleRoutingComplete`, because the component handles a routing indicator, not a general animation.

Reply:
resolved in bbd46c0eb8b037a66c32dc5968362425dd18a901